### PR TITLE
dt for g2 and bizcharts

### PR DESCRIPTION
**Pull request by Leannechn**
Please fill in this template.

- [ y ] Use a meaningful title for the pull request. Include the name of the package modified.
- [ y ] Test the change in your own code. (Compile and run.)
- [ y ] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [ y ] Avoid common mistakes(checked)
- [ y ] run tsc success

If adding a new definition:
- [ y ] The package does not provide its own types, and you can not add them.
- [ y ] It is for an NPM package, and match the name. 
- [ y ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [ y ] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

**Comment by Congee** [comment]
@Leannechn Hi, any updates?

**Comment by Leannechn** [comment]
I don't know how to solve the error, it prompts another other project types error.
My addtion is g2 and bizcharts, not `react-redux-epic`. can you help me ? what should I do ....
`Error: In /Users/yixin/works/vis/DefinitelyTyped/types/react-redux-epic/package.json: Dependency rxjs not in whitelist.`
image

File: types/g2/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,674 @@
...
+  const Animate: Animate;
+  const Util: Util;
+  const Shape: Shape;
+
+  class Global {
+    setTheme(option: 'default' | 'dark'): void;
+    version: string;
+    trackable: boolean;
+    animate: boolean;
+    snapArray: Array<number>;
```
Comment (Congee) [inline types/g2/index.d.ts]: @Leannechn I got the same error as Travis CI complains, but different from the Error in your image. I think there are 2 issues needed to be solved.
1. checkout to HEAD in your repo https://github.com/Leannechn/DefinitelyTyped and run the test again. Hope you get the same error, which has nothing to do with react-reduc-epic, as from Travis CI.
2. the index.d.ts file from types/g2 doesn't conform to what DefinitelyTyped requires. Please make the linter happy before makeing a pull request.

I cloned your code from you repo, and played around for a while. Found some issues.
```
> definitely-typed@0.0.1 test /tmp/DefinitelyTyped
> node node_modules/types-publisher/bin/tester/test.js --run-from-definitely-typed "types/g2"

Clean data
Clean logs
Clean output
# Typing Publish Report Summary
Started at Mon, 26 Mar 2018 03:38:18 GMT
Found 4162 typings folders in /tmp/DefinitelyTyped/types
Error: At 21:16 : Expected /<([^>]+)> */
    at Object.parseHeaderOrFail (/tmp/DefinitelyTyped/node_modules/definitelytyped-header-parser/index.js:47:15)
    at /tmp/DefinitelyTyped/node_modules/types-publisher/src/lib/definition-parser.ts:93:3
    at Generator.next (<anonymous>)
    at fulfilled (/tmp/DefinitelyTyped/node_modules/types-publisher/bin/lib/definition-parser.js:4:58)
    at <anonymous>
Error: Parsing failed.
    at fail (/tmp/DefinitelyTyped/node_modules/types-publisher/src/util/util.ts:320:11)
    at ChildProcess.child.on (/tmp/DefinitelyTyped/node_modules/types-publisher/src/util/util.ts:308:6)
    at ChildProcess.emit (events.js:180:13)
    at finish (internal/child_process.js:768:14)
    at process._tickCallback (internal/process/next_tick.js:112:11)
npm ERR! Test failed.  See above for more details.
```
Take a look at the error log, the test expected `/<([^>]+)> */`. What it really wants is a valid type annotation.
So I ran the linter, `npm run lint g2`. It complained `Array<T>` should be `T[]`. I corrected it, and ran `npm test types/g2` again, the error of this line was gone.

Hope this helps.

**Comment by Leannechn** [comment]
I temporarily shut down this PR, and wait for me to fix it , and we need to improve it. You are nice~

**Pull request closed without merging**